.rspec

```diff
@@ -1,3 +1,3 @@
 --color
 --tag ~server
---exclude spec/e2e/**/*
```

Comment:
It seems we are not running e2e specs on codeship till date. And also general specs and e2e specs cannot be loaded at the same time, due to a Json custom matcher conflict. So will be running these folders alone separately.